### DISCUSSION TITLE
Create A Flatpak for linux.

### DISCUSSION THREAD

**Discussion by DitherWither** [atom-ide-community/atom#432] [Ideas] [answered]
- Installing .rpm and .deb from the internet is a bad idea. They can cause issues with the system. Flatpaks can't break the system at all.
- Flatpaks are distro independent so maintaining builds for 3 different packaging formats wouldn't be necessary. It would also add better support for distros with immutable filesystems(fedora silverblue, steamos, etc).

Link to documentation:  [Introduction to Flatpak - Flatpak Documentation](https://docs.flatpak.org/en/latest/introduction.html)

**Comment by aminya** [comment, marked as answer]
Pull requests are welcome.

**Reply by DitherWither** [reply]
Where do I add the flatpak build though? This is my first time on large project.

**Reply by aminya** [reply]
Maybe under `./scripts/flatpack`

**Reply by DitherWither** [reply]
Will look into it. Thanks

**Comment by mauricioszabo** [comment]
We're trying to move to `electron-builder`, that may make flatpak easier as it's one of the supported targets
